### ISSUE TITLE
Sol Git - First community Solana token on GitHub.

### ISSUE THREAD

**Issue by GitHubSolana** [solana-labs/solana-web3.js#3272]
The first ever token on launched on the Official Solana GitHub Repo. All communcation will be made within this thread.

No TG. I will pay dex and give all updates via the comments.

The first comment will contain the CA/PF link.

$SOLGIT

**Comment by GitHubSolana** [comment]
https://pump.fun/C6FSRsnAKeu7jSGecQTuaVgBzWkBKceh74WHEJTCpump Live!

**Comment by Tottolin** [comment]
new meta?

**Comment by pnlgod** [comment]
We should use this thread a bit more fam

**Comment by Lockedinagency** [comment]
i love the idea 
but might be hard to get more eyes , we should do tg and gitbut and tell people to come in here

**Comment by Solmatt** [comment]
This could send hard

**Comment by kostia07** [comment]
I understand this possibly needs a twitter community or smth like that. But TG? For what?) All chads are on X. All devs are on Git.

**Comment by Tottolin** [comment]
we need more activity here

**Comment by Lockedinagency** [comment]
X community is a good idea and will bring the eyes the project deserve

**Comment by CryptoDer** [comment]
potential is huge

**Comment by notacryptobro** [comment]
Time to connect in here, everything looks promising, let the community talk!

**Comment by CryptoDer** [comment]
Please be the new meta 🙏

**Comment by CryptoDer** [comment]
> The first ever token on launched on the Official Solana GitHub Repo. All communcation will be made within this thread.
> 
> No TG. I will pay dex and give all updates via the comments.
> 
> The first comment will contain the CA/PF link.
> 
> $SOLGIT

LFG

**Comment by Solmatt** [comment]
Cabal shakeout

**Comment by Spenny1412** [comment]
Hey guys this is super cool! I hope we can expand the conversation on here this is a true chance at creating a brand new meta!

**Comment by Spenny1412** [comment]
I am pretty confident this will catch on once we shake the pump traders out here

**Comment by osmanucal** [comment]
Really good idea!

**Comment by Solmatt** [comment]
> I am pretty confident this will catch on once we shake the pump traders out here

I agree

**Comment by Tottolin** [comment]
dev? any news? is this a new meta or what?

**Comment by Tottolin** [comment]
doesnt feel like new meta... noone is here lol

**Comment by CryptoDer** [comment]
We are here

**Comment by osmanucal** [comment]
> doesnt feel like new meta... noone is here lol

How impatient. It hasn't even been 10 minutes and no one is complaining when there isn't lol

**Comment by Grantnav** [comment]
This is very interesting lol just signed up

**Comment by Grantnav** [comment]
Is there a place we can chat? This feels a little hard to do

**Comment by Tottolin** [comment]
cant say it was a bad idea, maybe with a based dev this can be a new meta

**Comment by vanuzx** [comment]
If we want to beat the Jeets, we buy when they sell. They will have to take it high.

**Comment by Tottolin** [comment]
this could work on eth, i like the idea tbh. but i feel like i miss something

**Comment by cantfindworse** [comment]
this is fye

**Comment by Githubmemecoins** [comment]
Guys this is a new meta, check top holders still holding the dip, see you at 1M, this is next snoofi

**Comment by Grantnav** [comment]
Top holders in. I’m in

**Comment by Grantnav** [comment]
This thread is actually dope

**Comment by Grantnav** [comment]
Maybe we can hold a record for longest thread lol

**Comment by vanuzx** [comment]
https://x.com/vanuzx/status/1838394823880876050 check my tweet and please support

**Comment by Spenny1412** [comment]
Raid me guys!!! 

https://x.com/therealspenny14/status/1838398805667901540?s=46&t=IO5M0T4V6c78gjEIsdhThA

**Comment by vanuzx** [comment]
Dev still here? @GitHubSolana

**Comment by vanuzx** [comment]
Guys, since my Twitter account is new, I may receive spam. Could you follow me, please?

https://x.com/vanuzx i can raid

**Comment by Prisonersix** [comment]
what is the purpose of this project  ? where are the MEMES to share ? can we do Twitter raids from here  lol

**Comment by Grantnav** [comment]
This will melt faces. I can feel it

**Comment by cantfindworse** [comment]
we can do raids and use this thread for memes coordinating etc

**Comment by FireBreathingBug** [comment]
cool af

**Comment by Tottolin** [comment]
we can upload pictures here 
![GYNNIsaWsAAF5Hm](https://github.com/user-attachments/assets/f0ec6790-6300-4e5e-a33b-30d44bb632cc)

**Comment by cantfindworse** [comment]
im bullish, lets work

**Comment by Grantnav** [comment]
Heart this if you’re BULLISH

**Comment by Spenny1412** [comment]
This is legit sad these jeets can’t even give anything a chance lmao

**Comment by Spenny1412** [comment]
How we feeling guys

**Comment by Tottolin** [comment]
dev can u say something, what is next, what should we do

**Comment by vanuzx** [comment]
> How we feeling guys

Good, why? has just been started.

**Comment by Spenny1412** [comment]
> > How we feeling guys
> 
> Good, why? has just been started.

Yes sir I’m grinding on the x communities! Let’s go

**Comment by Grantnav** [comment]
Everyone relax. Just the beginning of something crazy

**Comment by Grantnav** [comment]
> How we feeling guys

This is sick

**Comment by Sweatgate** [comment]
Love this.

**Comment by Garstarwar** [comment]
Marketing. It’s good but guess what. Nobody else knows.

**Comment by Sweatgate** [comment]
Imagine fading this

**Comment by Spenny1412** [comment]
It is a really cool idea I’d love to see more people come chat in here!

**Comment by Sweatgate** [comment]
Yeah its gonna get flooded soon buyers in coming

**Comment by Grantnav** [comment]
> Marketing. It’s good but guess what. Nobody else knows. 

I'm guessing the marketing will come. Some big whales in this if you track the wallets👀

**Comment by daniels74** [comment]
Github is poppin with devs 😌

**Comment by Spenny1412** [comment]
We gonna be in good hands guys he’s right the wallets are insane

**Comment by odysfox69** [comment]
we just need a good community to be formed with solid holders and thats it

**Comment by Grantnav** [comment]
GitHub might collab👀👀

**Comment by vanuzx** [comment]
If you like the idea. get it and work for your bags. It will be enough for the whales to see this. It's a really good idea.

**Comment by Tottolin** [comment]
well im going to trust the process, something new, if doesnt work at we can always cto xD

**Comment by Spenny1412** [comment]
So do we have any golfers in this community or what

**Comment by Grantnav** [comment]
SOLGIT IS META

**Comment by CryptoDer** [comment]
Now we talking

**Comment by vanuzx** [comment]
After a long time, I came across something really promising. I will hold it and support it

**Comment by Garstarwar** [comment]
> We gonna be in good hands guys he’s right the wallets are insane

Track the wallets. Were they given supply or did they actually buy in. 

Why am I asking? Most of those big wallets have copy traders. As soon as they purchased the copy traders would have purchased. We’d be in the millions if it’s huge whales.

**Comment by Sweatgate** [comment]
Like this idea its more of a low key community vibe almost nostalgic

**Comment by trippyfish** [comment]
genius

**Comment by Githubmemecoins** [comment]
> If you like the idea. get it and work for your bags. It will be enough for the whales to see this. It's a really good idea.

What can we do for help?

**Comment by Githubmemecoins** [comment]
Top holder are holding like crazy...so bullish

**Comment by vanuzx** [comment]
> > We gonna be in good hands guys he’s right the wallets are insane
> 
> Track the wallets. Were they given supply or did they actually buy in.
> 
> Why am I asking? Most of those big wallets have copy traders. As soon as they purchased the copy traders would have purchased. We’d be in the millions if it’s huge whales.

The best idea is to eliminate the jeets first.

**Comment by Spenny1412** [comment]
It really is the best idea the god candle after that will be glorious

**Comment by vanuzx** [comment]
> > If you like the idea. get it and work for your bags. It will be enough for the whales to see this. It's a really good idea.
> 
> What can we do for help?

https://x.com/vanuzx/status/1838403440906551343 RAID

**Comment by Tottolin** [comment]
i also posted on x 
https://x.com/casasfabiangbp/status/1838403474884558984

**Comment by Garstarwar** [comment]
> > > We gonna be in good hands guys he’s right the wallets are insane
> > 
> > 
> > Track the wallets. Were they given supply or did they actually buy in.
> > Why am I asking? Most of those big wallets have copy traders. As soon as they purchased the copy traders would have purchased. We’d be in the millions if it’s huge whales.
> 
> The best idea is to eliminate the jeets first.

I said Track the wallets and see if they actually purchased or someone gave them the tokens. 

That has nothing to do with the idea of Jeets.

**Comment by Spenny1412** [comment]
How is everyone’s evening going?

**Comment by vanuzx** [comment]
> > > > We gonna be in good hands guys he’s right the wallets are insane
> > > 
> > > 
> > > Track the wallets. Were they given supply or did they actually buy in.
> > > Why am I asking? Most of those big wallets have copy traders. As soon as they purchased the copy traders would have purchased. We’d be in the millions if it’s huge whales.
> > 
> > 
> > The best idea is to eliminate the jeets first.
> 
> I said Track the wallets and see if they actually purchased or someone gave them the tokens.
> 
> That has nothing to do with the idea of Jeets.

Even so, it's okay. If they wanted to sell, they would sell now.

**Comment by Garstarwar** [comment]
> > > > > We gonna be in good hands guys he’s right the wallets are insane
> > > > 
> > > > 
> > > > Track the wallets. Were they given supply or did they actually buy in.
> > > > Why am I asking? Most of those big wallets have copy traders. As soon as they purchased the copy traders would have purchased. We’d be in the millions if it’s huge whales.
> > > 
> > > 
> > > The best idea is to eliminate the jeets first.
> > 
> > 
> > I said Track the wallets and see if they actually purchased or someone gave them the tokens.
> > That has nothing to do with the idea of Jeets.
> 
> Even so, it's okay. If they wanted to sell, they would sell now.

I’m not worried about them selling. I just wanted to know if they purchased. That would be bullish. Pretending like they did so new people can feel bullish and ape. That’s treacherous.

**Comment by Tottolin** [comment]
so, start posting on X guys

**Comment by vanuzx** [comment]
We need whales to defeat these jeets.

**Comment by Spenny1412** [comment]
I had been posting on our x but now getting slandered for not only using git hub

**Comment by Prisonersix** [comment]
if this post takes off with thousands of comments. Will github drive traffic here ?

**Comment by Spenny1412** [comment]
Good question

**Comment by Sweatgate** [comment]
Haha jeets cant win im to lazy to watch wallets but everything is on chain. Gg

**Comment by vanuzx** [comment]
> if this post takes off with thousands of comments. Will github drive traffic here ?

I have no idea. But looks good

**Comment by Grantnav** [comment]
> if this post takes off with thousands of comments. Will github drive traffic here ?
> 
> 

I think so

**Comment by Spenny1412** [comment]
Looks like last jeets finishing up probable god candle coming

**Comment by Tottolin** [comment]
I'm starting to like this, just like the old days when there was no Telegram or any of those social networks, and we used forums to communicate.

**Comment by Garstarwar** [comment]
Well it shows we have 20 participants here. We need 100’s then thousands.

**Comment by toycatalyst** [comment]
![solgit](https://github.com/user-attachments/assets/1a4da44c-71aa-4fdc-8e5e-8f9dec424575)

**Comment by manimal17** [comment]
This is going to be a gold mine, let the jeets leave its a new coin and then once were all holding this thing will fly.... whole different community here

**Comment by Grantnav** [comment]
Jeets are starting to get shaken out. This could do millions. I think the dev is just waiting

**Comment by Spenny1412** [comment]
They should be out really soon usually takes about an hour

**Comment by Garstarwar** [comment]
Everybody commenting here follow @GitHubSolana

**Comment by Tottolin** [comment]
how can we bring more people to github?

**Comment by manimal17** [comment]
Anyone who knows devs knows that github is massive in the dev world... this just changed the game

**Comment by vanuzx** [comment]
I think if the dev thought of this, he was planning something.

**Comment by Grantnav** [comment]
> Everybody commenting here follow @GitHubSolana 

Yep go follow 🔥

**Comment by Grantnav** [comment]
I think we're early to something special here guys. I can feel it

**Comment by vanuzx** [comment]
I think this account is dev's smurf account. I believe you are a very solid person.

**Comment by Spenny1412** [comment]
I think it’s special too nice community forming in here too

**Comment by vanuzx** [comment]
I'm too sleepy. I'm going to sleep and I don't plan on selling what I have for a long time. This is NEW META. I truly believe this.

**Comment by Spenny1412** [comment]
If you have a minute guys go smash the rocket on dexscreener for more exposure

**Comment by Tottolin** [comment]
![QmPZSJyx9kWPMstftKQnkW3ycgHkL52f3mQBW2MZ3AYsyS](https://github.com/user-attachments/assets/e10467f4-26f9-4580-80f9-e1fe498f2087)

We can send memes here, idk, I’m just going to hold this. This is the kind of project that's worth the risk

**Comment by Spenny1412** [comment]
We have 39 members on x now not bad for an hour

**Comment by manimal17** [comment]
it looks like the top holder bought in also, thats huge!

**Comment by Tottolin** [comment]
LETS CHANGE OUR PFP FOR SOME GITHUB MEME, THIS CAN ACTUALLY BE A CULT

**Comment by Githubmemecoins** [comment]
Yes top holders holding like a crazy, jeeters will cry at 1 M

**Comment by Spenny1412** [comment]
![IMG_9640](https://github.com/user-attachments/assets/2e67544d-5f4b-4d18-833f-58f2e919d80d)


CHAD

**Comment by Prisonersix** [comment]
> Jeets are starting to get shaken out. This could do millions. I think the dev is just waiting

waiting for what  ?

**Comment by Grantnav** [comment]
> > Jeets are starting to get shaken out. This could do millions. I think the dev is just waiting
> 
> 
> 
> waiting for what  ?

For the jeeted to get out

**Comment by manimal17** [comment]
> ![IMG_9640](https://private-user-images.githubusercontent.com/132631550/370108471-2e67544d-5f4b-4d18-833f-58f2e919d80d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxNDU5MzAsIm5iZiI6MTcyNzE0NTYzMCwicGF0aCI6Ii8xMzI2MzE1NTAvMzcwMTA4NDcxLTJlNjc1NDRkLTVmNGItNGQxOC04MzNmLTU4ZjJlOTE5ZDgwZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyNFQwMjQwMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jYTM3MzhmZTVmNGU3YTFjMDE1OWQ4ODI0NTNlNzZkMDJlOWE3OWU0ZGNiNGI4OGY1OTYwYjkwNjI1MjcwYWMxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.jhYm9e0MksPwBLVNA9AQO_ODbuYzG6S2Lf5N-uZjkaI)
> 
> CHAD


holy chad, thats reckless but legendary at the same time

**Comment by Spenny1412** [comment]
> > ![IMG_9640](https://private-user-images.githubusercontent.com/132631550/370108471-2e67544d-5f4b-4d18-833f-58f2e919d80d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxNDU5MzAsIm5iZiI6MTcyNzE0NTYzMCwicGF0aCI6Ii8xMzI2MzE1NTAvMzcwMTA4NDcxLTJlNjc1NDRkLTVmNGItNGQxOC04MzNmLTU4ZjJlOTE5ZDgwZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyNFQwMjQwMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jYTM3MzhmZTVmNGU3YTFjMDE1OWQ4ODI0NTNlNzZkMDJlOWE3OWU0ZGNiNGI4OGY1OTYwYjkwNjI1MjcwYWMxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.jhYm9e0MksPwBLVNA9AQO_ODbuYzG6S2Lf5N-uZjkaI)
> > CHAD
> 
> holy chad, thats reckless but legendary at the same time

LFG!

**Comment by Tottolin** [comment]
lets try this, something new, i was so bored watching all this dogs and elephant tokens, lets try something new guys.

who know..

**Comment by manimal17** [comment]
we just need someone to market this thing big time

**Comment by Spenny1412** [comment]
Chart really getting exhausted reversal soon

**Comment by manimal17** [comment]
we need to start doing raids or something to push this up people need to see this

**Comment by Grantnav** [comment]
Lol how do you change pfp

**Comment by Githubmemecoins** [comment]
![Uploading OIG2.5JUFGmD_RruH.jpeg…]()

**Comment by Tottolin** [comment]
click on ur pfp and change it

**Comment by toycatalyst** [comment]
![anons_solgit](https://github.com/user-attachments/assets/02c7eae0-73eb-40a4-8adc-42cd3bc82bff)

**Comment by Githubmemecoins** [comment]
![OIG2 5JUFGmD_RruH](https://github.com/user-attachments/assets/7f46058f-6fcb-4ce2-add6-e706d1c110a8)

**Comment by Garstarwar** [comment]
> ![IMG_9640](https://private-user-images.githubusercontent.com/132631550/370108471-2e67544d-5f4b-4d18-833f-58f2e919d80d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxNDYwMDUsIm5iZiI6MTcyNzE0NTcwNSwicGF0aCI6Ii8xMzI2MzE1NTAvMzcwMTA4NDcxLTJlNjc1NDRkLTVmNGItNGQxOC04MzNmLTU4ZjJlOTE5ZDgwZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyNFQwMjQxNDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMDQyYjk4YzliMDkzNGVhNzIxYzJmZmM4NjQ1MjY5ZjVlN2Y3ZTJiZmIwYzBjNTY2N2E0ODVjM2FlMzhjZGQwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.WZKiYLijIbIiQtB3785IhBMB_b6m2915oKTXQfvl6g4)
> 
> CHAD

Now look at the rest of the wallet and see there activity. I always look to see if they’re a swing trader. 

Small holders miss out because they don’t ape big early so they’re waiting on gains. 

People like that see gains quick. That’s 52X the thousands of our marketcap 52X50 2600.

**Comment by Spenny1412** [comment]
Beyond me why anyone would sell at this level

**Comment by toycatalyst** [comment]
![image](https://github.com/user-attachments/assets/9925500f-20a8-4160-8783-0d9e8a3c1f94)

**Comment by Spenny1412** [comment]
Well guys it’s in your hands now I have to sleep keep commenting and chatting hope yall have a great night!

**Comment by manimal17** [comment]
everyone looks like theyre jeeting out just hold this will eventually be huge

**Comment by Githubmemecoins** [comment]
Tomorrow i will make more, i need to sleep, pretty sure this will go up

![OIG1 xWQHsytGe](https://github.com/user-attachments/assets/d4980a14-6db8-4398-99ff-f7a0736d79ad)

**Comment by Grantnav** [comment]
Top holders still holding

**Comment by Xapaxbt** [comment]
hodl

**Comment by Garstarwar** [comment]
> Beyond me why anyone would sell at this level

They’re looking for a pump or quick signs of activity. 

If price drops drastically it’s the floor. People buying from the top can’t dump the chart.

**Comment by Garstarwar** [comment]
> Top holders still holding

I’m going look at this.

**Comment by toycatalyst** [comment]
> we need to start doing raids or something to push this up people need to see this

Agree, post some raid links and we'll like and share

**Comment by Spenny1412** [comment]
Its so weird that zero top holders sold and the chart looks like this so confusing lol

**Comment by manimal17** [comment]
if we raid this it could potentially go to the moon, github is huge

**Comment by Grantnav** [comment]
> Its so weird that zero top holders sold and the chart looks like this so confusing lol

Makes no sense. I've been tracking. Where's the dev?

**Comment by Spenny1412** [comment]
> > Its so weird that zero top holders sold and the chart looks like this so confusing lol
> 
> Makes no sense. I've been tracking. Where's the dev?

Wish I knew man haha it’s the most confusing thing ever

**Comment by manimal17** [comment]
i guess the dev left? no clue how this works other than to comment

**Comment by Githubmemecoins** [comment]
I bought more and im going to sleep, im still really bullish

**Comment by Garstarwar** [comment]
> Top holders still holding

Okay a few of the top holders are from split bags.

**Comment by Grantnav** [comment]
Like it

https://x.com/thegnav/status/1838413184220176541?s=46&t=UEB_WMtZkqwNoNKZifsE4A

**Comment by Spenny1412** [comment]
![IMG_9642](https://github.com/user-attachments/assets/b072f41e-d958-423c-b246-65b9efe86ef5)


Wawa dev liking my tweet lets go!

**Comment by FineSprayPaintArt** [comment]
![Solgit 1](https://github.com/user-attachments/assets/adc14cee-fa47-4bbf-953b-816da55eeef3)
Not a bad idea to do this.  TIme to get people on board.

**Comment by manimal17** [comment]
> ![Solgit 1](https://private-user-images.githubusercontent.com/165910574/370113683-adc14cee-fa47-4bbf-953b-816da55eeef3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxNDczMzMsIm5iZiI6MTcyNzE0NzAzMywicGF0aCI6Ii8xNjU5MTA1NzQvMzcwMTEzNjgzLWFkYzE0Y2VlLWZhNDctNGJiZi05NTNiLTgxNmRhNTVlZWVmMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyNFQwMzAzNTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMGUwMzViNmFjMGVhYjczODA2MzZlNjdmNTJiNTI5MDU4MjAxMTRmMTFlODM2ZWMxZjVlOTljMDYwNjczZDBjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.u53hyuL-RSCpcXz2ca2g61cnmYJ_SmM81UCZhs-NyOU) Not a bad idea to do this. TIme to get people on board.

LFG

**Comment by manimal17** [comment]
Some big buys just came in lets go! Lets push this thing!

**Comment by Spenny1412** [comment]
It’s such a good meta we just keep it flowing in here

**Comment by FineSprayPaintArt** [comment]
I just reposted on my account.  I have few friends that follow along and might like theentry.  Lets see what happens
![Solgit 5](https://github.com/user-attachments/assets/8f7e90c8-f81e-4191-8756-bb145ab6649b)

**Comment by manimal17** [comment]
i get the github idea, but maybe we should market this heavy on X as well

**Comment by FineSprayPaintArt** [comment]
It has to be marketed there of course.  99.95% of Crupto Twitter is not on git hub

**Comment by manimal17** [comment]
with a solid raider here this could absolutely go crazy, top holders still holding

**Comment by FineSprayPaintArt** [comment]
Whales hold, I hold.  Seen other coins blast off from this kind of pull back. Its actually fun buying in more here to be rewarded later.

**Comment by manimal17** [comment]
> Whales hold, I hold. Seen other coins blast off from this kind of pull back. Its actually fun buying in more here to be rewarded later.

CHAD

**Comment by Grantnav** [comment]
I know if they get good raiders on this it will blow up

**Comment by Githubmemecoins** [comment]
https://x.com/h2ocoinproject3/status/1838415683803038159?t=wiGgs135A8-DsVy0ajtkQQ&s=19

**Comment by FineSprayPaintArt** [comment]
Raid some community posts 
https://x.com/Windswept1111/status/1838416123135398150

**Comment by Spenny1412** [comment]
Anyways I’m legit going to bed now haha keep it going guys! The more talk in here the better

**Comment by YomayObro** [comment]
The developers will be making a mistake if they don't let the project grow, if they don't let this token become the new meta

**Comment by FineSprayPaintArt** [comment]
Im just an Artist...

**Comment by manimal17** [comment]
Everyone slam the rocket on dexscreener so we get more eyes as well

We legit should start a tg for this at some point, theres way more functionality and traffic

keep meme posting in here and the tg for raids

**Comment by vanuzx** [comment]
> Everyone slam the rocket on dexscreener so we get more eyes as well
> 
> We legit should start a tg for this at some point, theres way more functionality and traffic
> 
> keep meme posting in here and the tg for raids

The logic is to use this place anyway. No TG

**Comment by FineSprayPaintArt** [comment]
The coin launches without a TG and explains no TG.  We get people in this chat and watch it send.

**Comment by manimal17** [comment]
yea but theres no bots for raids, no bots for filtering, youre missing functionality for when the jeets come into this

**Comment by FineSprayPaintArt** [comment]
Its a first mover.  We minimize the process for communication.  We use X to get people here and we do what we do.  MEME and RAID!
![SOlgit 4](https://github.com/user-attachments/assets/76176e68-0da7-4aa4-ba92-78a1ebc79030)

**Comment by daniels74** [comment]
Holy fucking shit were going back up 😭

**Comment by manimal17** [comment]
people are buying, lets keep raiding, and lets get some traction here so i can go to bed myself

**Comment by FineSprayPaintArt** [comment]
Even the icon on the DEX is dope!

**Comment by manimal17** [comment]
everything about this is dope we just need to raid, this is the start of the ultimate cult

**Comment by Grantnav** [comment]
We’re moving👀

**Comment by manimal17** [comment]
someones working on it LFG

**Comment by Spenny1412** [comment]
Top holders still holding 

chads

**Comment by Spenny1412** [comment]
Anyone doing anything crazy tonight or what I’m about to get baked lol

**Comment by Spenny1412** [comment]
Welp top holder gone

**Comment by manimal17** [comment]
wowwww

**Comment by Spenny1412** [comment]
All good he was a non chad

**Comment by Funnycoins80** [comment]
This is a real meta Nice patience is key on this one.

**Comment by Spenny1412** [comment]
Patience and you might be part of a meta starting coin. Life changing chance

**Comment by GatorRabbit** [comment]
What up

**Comment by Spenny1412** [comment]
> What up

Hey man I’m smoking a bowl watching some tv how about you haha

**Comment by Grantnav** [comment]
I know it's a CTO but it would be nice to have the founder in here talking to the community

**Comment by GatorRabbit** [comment]
Already did that, I don't watch TV. Just surfing PF

**Comment by GatorRabbit** [comment]
> I know it's a CTO but it would be nice to have the founder in here talking to the community

He could be sleeping

**Comment by GatorRabbit** [comment]
> > What up
> 
> Hey man I’m smoking a bowl watching some tv how about you haha

Already did that, I don't watch TV. Just surfing PF

**Comment by Grantnav** [comment]
Well we're pumping again

**Comment by zTonydaddyz** [comment]
I cant wait this alpha to get high.

**Comment by Lukethegee** [comment]
Yooooo let's send this hahahaha

**Comment by GatorRabbit** [comment]
> Yooooo let's send this hahahaha

Just takes a little time, tomorrow will be big

**Comment by Lukethegee** [comment]
Yeah I'm holding my bag brooooo!!!!

**Comment by Garstarwar** [comment]
Well at least we have new people coming in. Tell Dev amp up the marketing. Send Erik Stevenson some tokens and tell him it’s new meta and boom.

**Comment by Garstarwar** [comment]
> I know it's a CTO but it would be nice to have the founder in here talking to the community

Damn I didn’t know it was a CTO. These lazy as founders man. Like to sit in the background while we the community figure shit out.
